rest/server_context.go

```diff
@@ -358,14 +358,12 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	var importDocs, autoImport bool
 	switch config.ImportDocs {
-	case nil, false:
-	case true:
-		importDocs = true
+	case nil:
 	case "continuous":
 		importDocs = true
```

Comment:
We can remove 'importDocs' here, right?

Reply:
Yes, importDocs is removed since it was combined w/ the `autoImport` variable